tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es2018",
     "module": "commonjs",
-    "lib": ["es2018"],
+    "lib": ["es2018", "dom"],
```

Comment:
Wasnt this needed by the gui? as you have excluded it to a separate repo, i think you could revert this.